build.ps1

```diff
@@ -48,49 +45,6 @@ if (!(Test-Path $ToolPath)) {
     New-Item -Path $ToolPath -Type directory | out-null
 }
 
-###########################################################################
```

Comment:
Should resolve https://github.com/akkadotnet/akka.net/issues/3592 now that we're not downloading and installing `dotnet` now